apps/chat/src/components/Chat/SystemPrompt.tsx

```diff
@@ -120,14 +135,6 @@ export const SystemPrompt: FC<Props> = ({
     }
   }, [content]);
 
-  useEffect(() => {
```

Comment:
Are you sure that this should be removed?

Reply:
yes, I do